templates/default/env.erb

```diff
@@ -1,5 +1,6 @@
 <% %w(
 	buddy_check
+  configs
```

Comment:
But yeah, the others are all tabs 
![screenshot 2016-10-19 16 21 07](https://cloud.githubusercontent.com/assets/20174001/19522601/1e54b154-9618-11e6-9450-d87834cd2b4b.png)